logx/gumph.py

```diff
@@ -132,6 +135,11 @@ def __getattr__(self, method, *args, **kwargs):
         try:
             self.get_handler(logger_name)
         except KeyError:
-            self.add_std_handler(logger_name)
+            try:
```

Comment:
Thinking maybe this logic should be handled within `get_handler` (via adding an option to get both top-level and non-top-level handlers). Seems a bit cleaner than nested try/catches. What do you reckon?

Reply:
Yeah, thanks - I agree. I wasn't particularly happy with it to start with.

Reply:
Wait, no - the reason I did that was so I could get the right the `logger_name` without having to change the public interface. I'll make another function that returns a tuple of the handler and logger name instead.

Reply:
Check out the new changes.

Reply:
Looks good!